doc/source/contributor/environments/ci-multinode.rst

```diff
@@ -203,3 +203,132 @@ instance:
       ls testdir
 
 If it shows the test file then the share is working correctly.
+
+Magnum
+======
+
+The Multinode environment has Magnum enabled by default. To test it, you will
+need to create a Kubernetes cluster. It is recommended that you use the
+specified Fedora 35 image, as others may not work. Download the image locally,
+then extract it and upload it to glance:
+
+.. code-block:: bash
+
+      wget https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/35.20220410.3.1/x86_64/fedora-coreos-35.20220410.3.1-openstack.x86_64.qcow2.xz
+      unxz fedora-coreos-35.20220410.3.1-openstack.x86_64.qcow2.xz
+      openstack image create --container-format bare --disk-format qcow2 --file --property os_distro='fedora-coreos' --property os_version='35' fedora-coreos-35.20220410.3.1-openstack.x86_64.qcow2 fedora-coreos-35 --progress
+
+Create a keypair:
+
+.. code-block:: bash
+
+      openstack keypair create --private-key ~/.ssh/id_rsa id_rsa
+
+Install the Magnum, Heat, and Octavia clients:
+
+.. code-block:: bash
+
+      pip install python-magnumclient
+      pip install python-heatclient
+      pip install python-octaviaclient
+
+Create a cluster template:
+
+.. code-block:: bash
+
+      openstack coe cluster template create test-template --image fedora-coreos-35 --external-network external --labels etcd_volume_size=8,boot_volume_size=50,cloud_provider_enabled=true,heat_container_agent_tag=wallaby-stable-1,kube_tag=v1.23.6,cloud_provider_tag=v1.23.1,monitoring_enabled=true,auto_scaling_enabled=true,auto_healing_enabled=true,auto_healing_controller=magnum-auto-healer,magnum_auto_healer_tag=v1.23.0.1-shpc,etcd_tag=v3.5.4,master_lb_floating_ip_enabled=true,cinder_csi_enabled=true,container_infra_prefix=ghcr.io/stackhpc/,min_node_count=1,max_node_count=50,octavia_lb_algorithm=SOURCE_IP_PORT,octavia_provider=ovn --dns-nameserver 8.8.8.8 --flavor m1.medium --master-flavor m1.medium --network-driver calico --volume-driver cinder --docker-storage-driver overlay2 --floating-ip-enabled --master-lb-enabled --coe kubernetes
```

Comment:
Using a wallaby image?

```suggestion
      openstack coe cluster template create test-template --image fedora-coreos-35 --external-network external --labels etcd_volume_size=8,boot_volume_size=50,cloud_provider_enabled=true,heat_container_agent_tag=yoga-stable-1,kube_tag=v1.23.6,cloud_provider_tag=v1.23.1,monitoring_enabled=true,auto_scaling_enabled=true,auto_healing_enabled=true,auto_healing_controller=magnum-auto-healer,magnum_auto_healer_tag=v1.23.0.1-shpc,etcd_tag=v3.5.4,master_lb_floating_ip_enabled=true,cinder_csi_enabled=true,container_infra_prefix=ghcr.io/stackhpc/,min_node_count=1,max_node_count=50,octavia_lb_algorithm=SOURCE_IP_PORT,octavia_provider=ovn --dns-nameserver 8.8.8.8 --flavor m1.medium --master-flavor m1.medium --network-driver calico --volume-driver cinder --docker-storage-driver overlay2 --floating-ip-enabled --master-lb-enabled --coe kubernetes
```

Reply:
This one's actually intentional, apparently there isn't a yoga one and wallaby works fine